src/components/WordGroupSelector/WordGroupSelector.js

```diff
@@ -133,19 +134,22 @@ function WordGroupSelector(props) {
   }
 
   return (
-    <div className="Background">
-      <div className="WordGroups">
+    <div className="background">
+      <div className="word-groups-display">
         {Object.keys(wordGroups).map((key, index) => (
-          <div onClick={() => handleClick(wordGroups[key][0], key, index)}>
-            <WordGroupIcon
-              name={key}
-              image={wordGroups[key][1]}
-              colored={cardColored[index]}
-            />
+          <div className="word-groups-margins">
+            <div onClick={() => handleClick(wordGroups[key][0], key, index)}>
+              <WordGroupIcon
+                name={key}
+                image={wordGroups[key][1]}
+                colored={cardColored[index]}
+                onClick={() => handleClick(wordGroups[key][0], key, index)}
+              />
+            </div>
```

Comment:
was this your fix to being able to click on the outside of it?

Reply:
yuh